www/search

```diff
@@ -1226,6 +1227,10 @@ else if ($term || $browse)
 
         echo "<i>No results were found.</i>";
 
+        if ($games_filtered == true) {
```

Comment:
It's marginally better to say `if ($games_filtered)` without `== true`

Reply:
Changed this.